src/data-card-advanced.tsx

```diff
@@ -0,0 +1,498 @@
+                            display: 'flex',
+                        })}
+                        style={{
+                            marginTop: marginTopButton,
+                            marginRight: '16px',
+                            position: 'relative',
+                            zIndex: '2',
+                        }}
+                    >
+                        {button}
+                    </div>
+                )}
+                <div
+                    style={{
+                        display: 'flex',
+                        flexDirection: 'row',
+                        marginTop,
+                        flexWrap: 'wrap',
+                        marginRight,
+                    }}
+                >
+                    {hasFooterImage && (
+                        <div
+                            style={{marginRight: '16px', zIndex: '0'}}
+                            className={sprinkles({alignItems: 'center', display: 'flex'})}
+                        >
+                            {footerImage}
+                        </div>
+                    )}
+
+                    {hasFooterText && (
+                        <div style={{maxWidth}} className={styles.footerText}>
+                            <Text2 truncate={footerTextLinesMax} regular>
+                                {footerText}
+                            </Text2>
+                        </div>
+                    )}
+                </div>
+                {hasButtonLink && (
+                    // eslint-disable-next-line jsx-a11y/no-static-element-interactions
+                    <div
+                        aria-label="link"
+                        tabIndex={-1}
+                        className={sprinkles({
+                            display: 'flex',
+                        })}
+                        onClick={(event) => event.stopPropagation()}
+                        style={{
+                            marginTop: isMobile ? marginTop : marginTopButton,
+                            position: 'relative',
+                            zIndex: '2',
+                            marginLeft: -12,
+                            marginRight: -12,
+                        }}
+                    >
+                        {buttonLink}
+                    </div>
+                )}
+            </div>
+        </div>
+    );
+};
+
+type ExtraTypeof = typeof StackingGroup;
+
+type TextAs = 'h1' | 'h2' | 'h3' | 'h4';
+
+interface DataCardAdvancedProps {
+    onPress?: () => void;
+    stackingGroup?: RendersNullableElement<typeof StackingGroup>;
+    headline?: string | RendersNullableElement<typeof Tag>;
+    pretitle?: string;
+    pretitleLinesMax?: number;
+    pretitleAs?: TextAs;
+    title?: string;
+    titleLinesMax?: number;
+    titleAs?: TextAs;
+    subtitle?: string;
+    subtitleLinesMax?: number;
+    description?: string;
+    descriptionLinesMax?: number;
+    extra?: Array<RendersNullableElement<ExtraTypeof>>;
```

Comment:
extra shouldnt be an array, just a `React.ReactNode` like we do in the rest of Mistica components. Take into account that you can pass any react construction there, even a list of elements inside a fragment if you need, but there is no point of supporting arrays here.

Reply:
It was resolved this way because the CMS staff was having difficulty with the original Card since they could not control what would be added to the card's extra property, with that the Advanced Data Card was developed that would have this control with the blocks, in this way it was It was discussed that an array of blocks would be allowed so that the CMS could build the component, when this component was idealized it was agreed with someone from Novum, where we don't remember names at the moment, that it would be an array of blocks to help everyone and maintain a component control.

Reply:
I don't see any technical reason for this to be an Array. If the CMS sends an array to the frontend, the frontend can use a afragment and add there all the items in the array, I don't see the problem.

Reply:
The reason why the array is needed is due to two factors, namely the specific styling of the organization of the elements which if set to just React.node will cause an error, while there is also the issue that the advanced-data-card can only accept these blocks advanced-slot PR specifics plus StackingGroup

https://github.com/Telefonica/mistica-web/pull/770